packages/interaction-tracking/src/InteractionZone.js

```diff
@@ -42,19 +62,42 @@ export function wrap(callback: Function): Function {
   }
 
   const wrappedContext = currentContext;
+  const id = ++executionID;
+
+  __onWrappedContextScheduled(wrappedContext, id);
 
   return (...args) => {
     const prevContext = currentContext;
     currentContext = wrappedContext;
     try {
+      __onWrappedContextStarting(currentContext, id);
       callback(...args);
     } finally {
+      __onWrappedContextEnded(currentContext, id);
       currentContext = prevContext;
     }
   };
 }
 
-export function startContinuation(context: ZoneContext | null): void {
+/**
+ * If you call this function then you are promising to call startContinuation
+ * with the returned SettableZoneContext exactly once.
+ */
+export function getSettableContext(): SettableZoneContext | null {
+  if (!__PROFILE__) {
+    return null;
+  }
+  const context = isInContinuation ? continuationContext : currentContext;
```

Comment:
Something about this seems wrong, but I might be getting your PR– and my updated branch– mixed up...but if `getSettableContext()` is called while in a continuation– and this function returns `continuationContext`– won't that cause an error the next time it's called?